packer/run_interfaces.go

```diff
@@ -0,0 +1,39 @@
+package packer
+
+import "github.com/hashicorp/hcl/v2"
+
+type GetBuildsOptions struct {
+	// Get builds except the ones that match with except and with only the ones
+	// that match with Only. When those are empty everything matches.
+	Except, Only []string
+}
+
+type BuildGetter interface {
+	// GetBuilds return all possible builds for a config. It also starts all
+	// builders.
+	// TODO(azr): rename to builder starter ?
+	GetBuilds(GetBuildsOptions) ([]Build, hcl.Diagnostics)
+}
+
+//go:generate enumer -type FixConfigMode
+type FixConfigMode int
+
+const (
+	Stdout FixConfigMode = iota
```

Comment:
are these going to be fix command flags?

Reply:
Not for now ( be they could become yes), I created this because `packer validate` and `packer fix` do almost the same thing except `validate` will show a diff of the config and `fix` will output the whole config file.